### DISCUSSION TITLE
Starstruck Badge

### DISCUSSION THREAD

**Discussion by peripheralmike** [Schweinepriester/github-profile-achievements#661] [Q&A]
Would really appreciate some help getting the Starstruck Badge - I already have 6 stars on my repo here https://github.com/PeripheralMike/Root-Android-Emulator 

any more would be a huge help. Thank you.

**Comment by ArielMAJ** [comment]
@PeripheralMike I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by peripheralmike** [reply]
Starred, thank you

**Reply by ArielMAJ** [reply]
Ty!

**Comment by guppy57** [comment]
Hey, just left you a star! If you like anything on my profile, feel free to leave one.

Cheers!

**Reply by ArielMAJ** [reply]
@guppy57  I left you some stars. Please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.
